README.md

```diff
@@ -12,45 +12,79 @@ NetBox Operator extends the Kubernetes API by allowing users to manage NetBox re
 # Getting Started
 
 ## Prerequisites
+
 - go version v1.22.0+
 - docker version 17.03+.
 - kubectl version v1.11.3+.
 - Access to a Kubernetes v1.11.3+ cluster.
 
-## Running and developing locally with the Go debugger
+## Running and developing locally
+
+### Running the NetBox Operator and NetBox on a local kind cluster
+
+Note: This requires Docker BuildKit.
+
+- Create kind cluster with a NetBox deployment: `make create-kind`
+- Port-forward NetBox: `kubectl port-forward deploy/netbox 8080:8080 -n default`
+- Open <http://localhost:8080> in your favorite browser and log in with the username `admin` and password `admin`, you will be able to access the local NetBox instance running in the kind cluster.
+- Deploy the NetBox Operator on the local kind cluster: `make deploy-kind`
 
-### Using a kind cluster
+### Running the NetBox Operator on your machine and NetBox on a local kind cluster
 
-Follow the instructions to bring up a locally running kind cluster, with NetBox and a NetBox operator running within it:
-- execute `make create-kind`
-- set the following environment varialbes:
+- Create kind cluster with a NetBox deployment: `make create-kind`
+- Port-forward NetBox: `kubectl port-forward deploy/netbox 8080:8080 -n default`
+- Open <http://localhost:8080> in your favorite browser and log in with the username `admin` and password `admin`, you will be able to access the local NetBox instance running in the kind cluster.
+- Open a new terminal window and export the following environment variables:
+
+    ```bash
+    export NETBOX_HOST="localhost:8080"
+    export AUTH_TOKEN="0123456789abcdef0123456789abcdef01234567"
+    export POD_NAMESPACE="default"
+    export HTTPS_ENABLE="false"
+    export NETBOX_RESTORATION_HASH_FIELD_NAME="netboxOperatorRestorationHash"
     ```
-    NETBOX_HOST="localhost:8080"
-    AUTH_TOKEN="0123456789abcdef0123456789abcdef01234567"
-    POD_NAMESPACE="default"
-    HTTPS_ENABLE="false"
-    NETBOX_RESTORATION_HASH_FIELD_NAME="netboxOperatorRestorationHash"
+
+- Run the NetBox Operator locally (in a new terminal window) `make install && make run`
```

Comment:
This should be in the same terminal as the export statements are run else they'll not be picked up